src/modules/StudyManager/components/List/container.js

```diff
@@ -52,6 +52,10 @@ export class List extends Component {
     this.persistFilters = this.persistFilters.bind(this);
   }
 
+  componentWillMount() {
+    this.props.activateModule();
```

Comment:
Why not to use ActiveModuleAwareContainerBuilder?

Reply:
because in these components needed information is already loaded, and we have enough information to decide which module should be highlighted

Reply:
@agackovka , so why not to handle such case in `ActiveModuleAwareContainerBuilder`? would be great to have a single place for the handling of the sidebar tab selection